lib/dicthf.gi

```diff
@@ -40,7 +41,14 @@ InstallMethod(SparseIntKey,"defaults to DenseIntKey",true,[IsObject,IsObject],
 ##  between different runs of {\GAP} and no reference to their absolute
 ##  values ought to be made.
 ##
+##  HashKeyWholeBag hashes all the contents of a bag, which is equivalent
+##  to passing 0 and -1 as the third and fourth arguments of HashKeyBag.
+##  Be aware that for many types in GAP (for example permutations), equal
+##  objects may not have identical bags, so HashKeyWholeBag may return
+##  different values for two equal objects.
+##
 BindGlobal("HashKeyBag",HASHKEY_BAG);
+BindGlobal("HashKeyWholeBag", {x,y} -> HASHKEY_BAG(x,y,0,-1));
```

Comment:
Nitpick: I find this name confusing... Indeed, also `HashKeyBag` is weird to me. How about `HashOfWholeBag`, `HashForWholeBag`, `HashWholeBag`?